src/components/user-list-test.html

```diff
@@ -0,0 +1,49 @@
+<link rel="import" href="../../node_modules/@banno/polymer/polymer.js">
+<link rel="import" href="../../node_modules/@banno/polymer/lib/elements/dom-repeat.js">
+<link rel="import" href="./user-profile.html">
+
+<dom-module id="user-list">
+  <style>
+  </style>
+  <template>
+    <h2> User list: </h2>
+    <dom-repeat items="{{users}}">
+      <template>
+        <div>
+          <user-profile 
+            id="{{item.id}}" 
```

Comment:
All of these properties should be one-way bound (`[[ ... ]]`) instead of two-way bound (`{{ ... }}`)
```suggestion
            id="[[item.id]]" 
```

About the only time you'll _ever_ use two way binding is when you have a text input that you want to update some model. That's why you're two-way binding in your create and edit user components. Here though (and just about everywhere else)? You'll want to one-way bind.